components/Ballot/Race.tsx

```diff
@@ -0,0 +1,147 @@
+import { FieldSet } from "components/FieldSet/FieldSet";
+import { VotingGuideNote } from "components/VotingGuide/VotingGuideNote/VotingGuideNote";
```

Comment:
I'm down with that...I actually have some stashed changes that converts every component to named exports and exports them all from an index file in the `components` directory.  How do you feel about that? Just one index at the /components root instead of index files in each /components/subdir ?

Reply:
sounds good to me